roles/app/templates/post-receive.j2

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-APP_PATH="$HOME/apps/openfoodnetwork"
+APP_PATH="$HOME/apps/ofn_america"
```

Comment:
Why change this? Its not really intended to be changed. Does it need to be a variable?